pkg/distro/packagesets/fedora/package_sets.yaml

```diff
@@ -0,0 +1,902 @@
+---
+cloud_base:  # not used directly; added for reference
+  include:
+    - "@Fedora Cloud Server"
+    - "chrony"  # not mentioned in the kickstart anaconda pulls it when setting the timezone
+    - "langpacks-en"
+  exclude:
+    - "dracut-config-rescue"
+    - "firewalld"
+    - "geolite2-city"
+    - "geolite2-country"
+    - "plymouth"
+
+qcow2: &qcow2
+  include:
+    # cloud_base
+    - "@Fedora Cloud Server"
+    - "chrony"  # not mentioned in the kickstart anaconda pulls it when setting the timezone
+    - "langpacks-en"
+
+    # qcow2
+    - "qemu-guest-agent"
+  exclude:
+    # cloud_base
+    - "dracut-config-rescue"
+    - "firewalld"
+    - "geolite2-city"
+    - "geolite2-country"
+    - "plymouth"
+
+ami:
+  <<: *qcow2
+
+oci:
+  <<: *qcow2
+
+openstack:
+  <<: *qcow2
+
+vhd:
+  include:
+    # cloud_base
+    - "@Fedora Cloud Server"
+    - "chrony"  # not mentioned in the kickstart anaconda pulls it when setting the timezone
+    - "langpacks-en"
+
+    # vhd
+    - "WALinuxAgent"
+  exclude:
+    # cloud_base
+    - "dracut-config-rescue"
+    - "firewalld"
+    - "geolite2-city"
+    - "geolite2-country"
+    - "plymouth"
+
+vmdk: &vmdk
+  include:
+    - "@Fedora Cloud Server"
+    - "chrony"
+    - "systemd-udev"
+    - "langpacks-en"
+    - "open-vm-tools"
+  exclude:
+    - "dracut-config-rescue"
+    - "etables"
+    - "firewalld"
+    - "geolite2-city"
+    - "geolite2-country"
+    - "gobject-introspection"
+    - "plymouth"
+    - "zram-generator-defaults"
+    - "grubby-deprecated"
+    - "extlinux-bootloader"
+
+ova:
+  <<: *vmdk
+
+iot_commit: &iot_commit
```

Comment:
Could you add a comment here about keeping this list in sync with the package list in pagure over here - https://pagure.io/fedora-iot/ostree/blob/main/f/fedora-iot-base.yaml

We have a comment over there about keeping osbuild/images up-to-date (https://pagure.io/fedora-iot/ostree/blob/main/f/fedora-iot-base.yaml#_76) but never made a similar comment in osbuild/images

Reply:
I wonder how much work it would be to have an integration test that compares the two and fail if they start to diverge. But obviously followup material!